cpp-terminal/base.cpp

```diff
@@ -6,13 +6,13 @@
 
 
 std::string Term::color(style value) {
-    return "\033[" + std::to_string(static_cast<unsigned int>(value)) + 'm';
+    return "\033[" + std::to_string((unsigned int)value) + 'm';
```

Comment:
We should not use C casts in C++. Let's keep using the `static_cast`.

Reply:
Ok sure!